api/controllers/blocks.js

```diff
@@ -56,7 +74,7 @@ BlocksController.getBlocks = function(context, next) {
 	// Remove filters with null values
 	filters = _.pickBy(filters, v => !(v === undefined || v === null));
 
-	return modules.blocks.shared.getBlocks(_.clone(filters), (err, data) => {
+	return _list(_.clone(filters), (err, data) => {
```

Comment:
That's the only place the function is used, it's easier to keep the logic of returning blocks in one place, so my suggestion is not to extract the _list logic into a separate function.

Reply:
This will require a big change in the unit tests. I will keep as it is for now.